nfv_sriov.md

```diff
@@ -0,0 +1,253 @@
+# NFV SRIOV
+
+This document describes how to deploy EDPM nodes with SRIOV.
+
+## Procedure Overview
+
+1. Configure kernel args and tuned parameters
+2. Configure SRIOV parameters
+3. Configure the networks with SRIOV interfaces of the EDPM nodes
+2. Configure Nova and Neutron parameters.
+
+In order to complete the above procedure, the `services` list of the
+`OpenStackDataPlaneNodeSet` CR needs to be edited.
+
+## OpenStackDataPlaneNodeSet services list
+
+EDPM nodes can be configured by creating an
+`OpenStackDataPlaneNodeSet` CR which the
+[dataplane-operator](https://openstack-k8s-operators.github.io/dataplane-operator)
+will reconcile to create OpenStackDataPlaneService resources
+when an `OpenStackDataPlaneDeployment` CR is created.
+These types of CRs have a `services` list like the following:
+
+```yaml
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+spec:
+  ...
+  services:
+    - bootstrap
+    - download-cache
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - ssh-known-hosts
+    - run-os
+    - reboot-os
+    - install-certs
+    - libvirt
+    - ovn
+    - neutron-ovn
+    - nova-custom-sriov
+    - neutron-sriov
+    - neutron-metadata
+```
+Only the services which are on the list will be configured.
```

Comment:
Please add a note about `OpenStackDataPlaneService` CRs, explaining that these are the services being created. We must avoid confusion with other "services" due to the overloaded terminology.

Reply:
done